content/docs/command-reference/repro.md

```diff
@@ -166,7 +166,7 @@ only execute the final stage.
 
 For simplicity, let's build a pipeline defined below. (If you want get your
 hands-on something more real, see this short
-[pipeline tutorial](/doc/tutorials/pipelines)). It takes this `text.txt` file:
+[pipeline tutorial](/doc/tutorials/dags)). It takes this `text.txt` file:
```

Comment:
> Line 169 in repro.md links to /doc/tutorial/dags and previously linked to /doc/tutorial/pipelines

Yeah both are wrong. I would even prefer `/doc/tutorial/pipelines` but the whole tutorial is gone now so you can just remove the whole parenthesis.

Reply:
Please check whether these links still exist anywhere else in docs.

Reply:
> you can just remove the whole parenthesis

Or find a better place to link to e.g. /doc/start/data-pipelines

Please decide whether it's best to change this sentence and link, or just remove it.

Reply:
I think it's better to have a tutorial here. `repro` is related to pipelines, and having an option to explore pipelines in detail looks helpful.